src/main/kotlin/frc/robot/subsystems/swerve/ModuleIO.kt

```diff
@@ -37,6 +37,8 @@ interface ModuleIO {
 
     fun setVoltage(volts: Double) {}
 
+    fun setIdleMode(isBreakMode: Boolean) {}
```

Comment:
Why not pass in NeutralModeValue instead of a boolean?

Reply:
Because the function should work for every motor type.

Reply:
Oh right, forgot we have a SparkMaxIO